### PR TITLE
Restore 3 action bindings to event dispatch sequence.

### PR DESCRIPTION
**Pull request by quickdudley**
Fixes unreported bug: restores functionality to the actions under View/Miscellaneous.

In PR#4440: the handlers for these actions had been attached under `WorldView::InputBindings` (before this they were under `KeyBindings`), but there was no control flow path to the bindings in that object from the main event loop.

### CODE REVIEW

File: src/WorldView.cpp [hunk 1]
```diff
@@ -1132,3 +1132,9 @@ vector3d WorldView::GetMouseDirection() const {
 		mouseDir = -mouseDir;
 	return (Pi::player->GetPhysRadius() * 1.5) * mouseDir;
 }
+
+void WorldView::HandleSDLEvent(SDL_Event &event) {
+	InputBindings.toggleHudMode->CheckSDLEventAndDispatch(&event);
+	InputBindings.increaseTimeAcceleration->CheckSDLEventAndDispatch(&event);
+	InputBindings.decreaseTimeAcceleration->CheckSDLEventAndDispatch(&event);
+}
```
Comment (impaktor) [inline src/WorldView.cpp]: github says you need to press RET here.

**Comment by laarmen** [comment]
We really need to have a clang-format flag day, because this PR is really hard to review *because* of the tool. That's a shame (and not the OP's fault, obviously)

**Comment by fluffyfreak** [comment]
I am assuming that the large number of diff changes are from the clang-format?
It's hard to tell what's actually different, ah well.

What name would you like to be known under in the `AUTHORS.txt` file after I merge this?

**Comment by quickdudley** [comment]
The large number of diff changes is indeed from clang-format: but fortunately my own changes are all in the first two commits and the last two commits are just formatting.

I would like to be known as Jeremy List in `AUTHORS.txt`

**Pull request merged**